api/core/v1alpha1/common_types.go

```diff
@@ -60,6 +60,24 @@ const (
 	ReasonPodNotDeleted  = "PodNotDeleted"
 )
 
+const (
+	// StoreOfflineConditionType represents the condition of the store offline process.
+	StoreOfflineConditionType = "Offlining"
```

Comment:
Offlining vs Offlined. I prefer Offlined to indicate the offline progress is done.